cogs/subscribers.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) idoneam (2016-2019)
+#
+# This file is part of Canary
+#
+# Canary is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Canary is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Canary. If not, see <https://www.gnu.org/licenses/>.
+
+# discord-py requirements
+import discord
+from discord.ext import commands
+from discord import utils
+import asyncio
+
+# URL access and parsing
+from bs4 import BeautifulSoup
+
+# Other utilities
+import os
+import re
+import pickle
+import feedparser
+import requests
+
+CFIA_FEED_URL = "http://inspection.gc.ca/eng/1388422350443/1388422374046.xml"
```

Comment:
It kinda makes more sense to have both the CFIA and metro API URL as constants at the top of this file instead of the metro API URL as a config value imo

Reply:
Sure thing!